indexer/indexer.go

```diff
@@ -154,6 +184,13 @@ func defaultBadgerOptions(
 	// filters will be immediately discarded from the cache).
 	opts.LoadBloomsOnOpen = false
 
+	// BlockSize sets the size of any block in SSTable. SSTable is divided into multiple blocks
+	// internally. We set each block to 1MB.
+	opts.BlockSize = 1 << blockLeftShift // 1MB
```

Comment:
If this has any relationship to the size of blocks in our mainchain, 1MB would not be enough as we support larger blocks.